jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -856,7 +856,7 @@ void java_bytecode_convert_classt::add_array_types(symbol_tablet &symbol_table)
     code_declt declare_cloned(local_symexpr);
 
     source_locationt location;
-    location.set_function(local_name);
+    location.set_function(local_symbol.pretty_name);
```

Comment:
This isn't a function name (`clone_symbol` refers to a function name in this case)